imagetagger/imagetagger/base/templates/base/markdown_guide.html

```diff
@@ -0,0 +1,137 @@
+                            <td>Heading 1<br>========</td>
+                            <td>
+                                <h1>Heading 1</h1>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>## Heading 2</td>
+                            <td>Heading 2<br>--------------</td>
+                            <td>
+                                <h2>Heading 2</h2>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>&gt; Blockquote</td>
+                            <td></td>
+                            <td>
+                                <blockquote>Blockquote</blockquote>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>A paragraph.<br><br> A paragraph after 1 blank line.</td>
+                            <td></td>
+                            <td>A paragraph.<br>A paragraph after 1 blank line.</td>
+                        </tr>
+                        <tr>
+                            <td>* List<br> * List<br> * List</td>
+                            <td>- List<br> - List<br> - List</td>
+                            <td>
+                                <ul>
+                                    <li>List</li>
+                                    <li>List</li>
+                                    <li>List</li>
+                                </ul>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>1. One<br> 2. Two<br> 3. Three</td>
+                            <td></td>
+                            <td>
+                                <ol>
+                                    <li>One</li>
+                                    <li>Two</li>
+                                    <li>Three</li>
+                                </ol>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>Horizontal Rule<br><br> -----------</td>
+                            <td>Horizontal Rule<br><br> ***********</td>
+                            <td>Horizontal Rule
+                                <hr>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>`Inline code` with backticks</td>
+                            <td></td>
+                            <td>
+                                <code>Inline code</code> with backticks
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>[Link](http://url.org)</td>
+                            <td></td>
+                            <td><a href="#">Link</a></td>
+                        </tr>
+                        <tr>
+                            <td>![Image](http://url/image.png)</td>
```

Comment:
The discussion about whether we want to allow external images or not seems still unresolved.